### DISCUSSION TITLE
Using VueJS Component inside a widget ?

### DISCUSSION THREAD

**Discussion by schlaumeier2023** [filamentphp/filament#9534] [Help]
Hi,

I am looking for a way to use a custom designed vuejs component inside of a widget in filament php. Do you guys have any idea how to do it?

**Comment by leandrocfe** [comment]
Maybe [this section](https://filamentphp.com/docs/3.x/support/assets) can help you

**Reply by schlaumeier2023** [reply]
I already have seen that section, however I was not able to solve my issue

**Reply by leandrocfe** [reply]
Why not [alpinejs](https://alpinejs.dev/) ?

**Reply by schlaumeier2023** [reply]
Because it is an existing project that we are using FilamentPHP where I need to use the current existing components from vueJS

**Reply by PrasadChinwal** [reply]
@schlaumeier2023 were you able to solve this? I want to also include a vuejs component in a livewire page in filament.
